source/solutions-library.txt

```diff
@@ -253,7 +253,7 @@ kick-start their projects.
 
          .. card::
             :headline: Elevate Flight Operations with Real-Time Analytics
-            :url: https://deploy-preview-222--docs-atlas-architecture.netlify.app/solutions-library/flightmanagement/
+            :url: /solutions-library/flightmanagement/
```

Comment:
(suggestion)
We might standardize on hyphens between words. This looks good as is, but we'll follow up tomorrow before we merge into Atlas Architecture Center.